src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+    }
+
+    public static void printList(){
+        System.out.println(YOUR_LIST);
+        for(int i=0; i<numItems; i++){
+            System.out.println("\t"+(i+1)+". "+ list[i]);
+        }
+        System.out.println(LINE_BREAK);
+    }
+    public static void addToList(String answer, TaskType taskType){
+        numItems++;
+        String taskDescription, time;
+        switch (taskType){
+        case TODO:
+            list[numItems-1] = new ToDo(answer);
+            break;
+        case DEADLINE:
+            taskDescription = answer.substring(0, answer.indexOf("/by"));
+            time = answer.substring(answer.indexOf("/by ")+3);
+            list[numItems-1] = new Deadline(taskDescription,time);
+            break;
+        case EVENT:
+            taskDescription = answer.substring(0, answer.indexOf("/at"));
+            time = answer.substring(answer.indexOf("/at ")+3);
+            list[numItems-1] = new Event(taskDescription, time);
+        }
+        System.out.println(LINE_BREAK);
+        System.out.println("\tAdded:" + list[numItems-1]);
+        System.out.println("\tNow you have "+numItems+(numItems>1?" tasks":" task")+" in the list :D");
+        System.out.println(LINE_BREAK);
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        int totalTasksDone=0;
+        String answer="";
+        System.out.println(LINE_BREAK);
+        System.out.println("\tHello! I'm\n"+ LOGO);
+        System.out.println("\tWhat can I do for you?");
+        System.out.println(LINE_BREAK);
+        while(true){
+            answer = in.nextLine();
+            if(answer.trim().equalsIgnoreCase("bye")){
+                break;
+            }
+            if(answer.trim().equalsIgnoreCase("list")){
+                printList();
+                continue;
+            }
+            if(answer.contains("done")){
+                answer = answer.trim();
+                String[] words = answer.split(" ");
+                if(words.length>1) {
+                    int valueToMarkDone = Integer.parseInt(words[1]);
+                    if(valueToMarkDone>0 && valueToMarkDone<=numItems) {
+                        if(!list[valueToMarkDone-1].isDone){
+                            totalTasksDone++;
+                        }
+                        list[valueToMarkDone - 1].markAsDone();
+                        System.out.println("\tAwesome! I've marked this task as done:");
+                        System.out.println("\t["+list[valueToMarkDone - 1]);
+                        System.out.println("\tOnly "+(numItems-totalTasksDone)+" to go! ;)");
+                    }
+                    else{
```

Comment:
I noticed that for a few cases of if-else statements, you formatted it like:
}
else{
I think it would be better to format it like this: "} else {" with the two ends of the curly braces on the same line as "else".